src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -40,304 +40,159 @@
  * suppressed nodes inside the AST tree.
  * </p>
  * <p>
- * Currently, filter supports the following checks:
+ * Currently, filter does not support checks derived from {@code AbstractJavadocCheck}
```

Comment:
User should not know about our inner class hierarchy.
It is better to just mention them here.

Reply:
Done. Now there is two sets of incompatible checks in the test `XpathRegressionTest`.
One for Javadoc checks, another for Java checks.